src/main/groovy/com/chrisgahlert/gradledcomposeplugin/tasks/AbstractDcomposeTask.groovy

```diff
@@ -136,7 +137,9 @@ class AbstractDcomposeTask extends DefaultTask {
     }
 
     protected String toJson(Object input) {
-        new JsonBuilder(input).toPrettyString()
+        // Strip the "rawValues" property since it cannot be "diffed" effectively.
```

Comment:
New client includes a `rawValues` property which cannot be tweaked and may contain information that changes in ways that makes change management difficult to do.